crates/shared/src/code_simulation.rs

```diff
@@ -101,6 +106,46 @@ impl TenderlyCodeSimulator {
         };
         self
     }
+
+    fn prepare_request(
+        &self,
+        call: CallRequest,
+        overrides: StateOverrides,
+        block: Option<u64>,
+    ) -> Result<SimulationRequest> {
+        Ok(SimulationRequest {
+            block_number: block,
+            network_id: self.network_id.clone(),
+            from: call.from.unwrap_or_default(),
+            to: call.to.unwrap_or_default(),
+            input: call.data.unwrap_or_default().0,
```

Comment:
any reason why we unwrap here? I assume you expect we always have this data so no point in handling this case?